website/versioned_docs/version-7.8.0/standalone.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Copied from revised `docs/standalone` to highlight the `presets: ["env"]` usage.